proprietary/tokens/src/components/amsterdam/alert.tokens.json

```diff
@@ -1,36 +1,42 @@
 {
   "amsterdam": {
     "alert": {
-      "background-color": { "value": "{amsterdam.color.yellow}" },
-      "border": { "value": "4px solid {amsterdam.color.yellow}" },
+      "border-width": { "value": "{amsterdam.border-width.xl}" },
+      "border-style": { "value": "solid" },
       "gap": { "value": "1rem" },
       "padding-block-start": { "value": "{amsterdam.spacing.inset.md}" },
       "padding-block-end": { "value": "{amsterdam.spacing.inset.md}" },
       "padding-inline-start": { "value": "{amsterdam.spacing.inset.lg}" },
       "padding-inline-end": { "value": "{amsterdam.spacing.inset.lg}" },
+      "close-button": {
```

Comment:
This is also not needed, we need to check why actionButton is applied to everything except Alert in Figma.

Reply:
It is in this PR, we’ll remove it again when applying Icon Button.